Content.Server/Stunnable/StunbatonSystem.cs

```diff
@@ -200,5 +204,14 @@ private void TurnOn(StunbatonComponent comp, EntityUid user)
             sprite.LayerSetState(0, "stunbaton_on");
             comp.Activated = true;
         }
+
+        private void SendPowerPulse(EntityUid target, EntityUid? user, EntityUid used)
+        {
+            RaiseLocalEvent(target, new PowerPulseEvent()
+            {
+                Used = used,
+                User = user
+            });
```

Comment:
```suggestion
            RaiseLocalEvent(target, new PowerPulseEvent()
            {
                Used = used,
                User = user
            }, false);
```

for just a directed event